src/components/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function useClassName<T>(state: T, classes: [T, string][]) {
+  return (classes.find(_class => _class[0] === state) || [])[1] || '';
+}
```

Comment:
Not sure if this is the right place for this. Maybe we should have a `useStyles` composable that contains style helper functions like this. And then you can rename this function like Ev recommended.

Reply:
renamed